env.template

```diff
@@ -75,8 +72,12 @@ SCOS_SENSOR_GIT_TAG="$(git describe --tags)"
 # SECURITY WARNING: generate unique key with `manage.py generate_secret_key`
 SECRET_KEY="$(python3 -c 'import secrets; print(secrets.token_urlsafe(64))')"
 
+# Signal analyzer selection/setup
 SIGAN_CLASS=TekRSASigan
 SIGAN_MODULE=scos_tekrsa.hardware.tekrsa_sigan
+USB_DEVICE=Tektronix
+DEVICE_MODEL=RSA507A
+BASE_IMAGE=ghcr.io/ntia/scos-tekrsa/tekrsa_usb:0.2.3
```

Comment:
Consider changing the 0.2.3 to `latest` to match scos-tekrsa README, but perhaps there is a need to keep it pinned to specific version?